client/my-sites/plugins/plans/index.tsx

```diff
@@ -44,9 +45,20 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 		);
 	}, [ dispatch, translate, selectedSite, breadcrumbs.length, intervalType ] );
 
-	const onSelectPlan = () => {
+	useEffect( () => {
+		const onResize = () => {
+			setIsDesktopDevice( isDesktop() || false );
+		};
+
+		onResize();
+		window.addEventListener( 'resize', onResize );
+
+		return () => window.removeEventListener( 'resize', onResize );
+	}, [] );
```

Comment:
There's a hook for this. https://github.com/Automattic/wp-calypso/blob/9e36d03760163ecfd883239ae6e8b17be3d4523f/client/my-sites/plans/jetpack-plans/product-grid/index.tsx#L115

Reply:
Thanks, I searched and didn't found it.

Reply:
Calypso is huge, reinventing the wheel is going to happen sometimes :slightly_smiling_face: